Realm/Swift/RLMSupport.swift

```diff
@@ -17,6 +17,7 @@
 ////////////////////////////////////////////////////////////////////////////
 
 import Realm
+import Realm.Private
```

Comment:
This means we'll be exposing everything, such as the dynamic API and the `RLMObjectBase` functions, to everyone who uses this -- not sure I love that.